.github/workflows/main.yml

```diff
@@ -30,43 +30,6 @@ jobs:
           python -m pip install --upgrade pip
           pip install -e .
 
-  pkg-test:
-    runs-on: ubuntu-latest
-    needs: pkg-install
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.6
-        uses: actions/setup-python@v1
-        with:
-          python-version: 3.6
-          architecture: x64
-      - name: Cache python modules
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pkg-deps-${{ hashFiles('requirements.txt') }}-${{ hashFiles('**/*.py') }}
-          restore-keys: |
-            ${{ runner.os }}-pkg-deps-${{ hashFiles('requirements.txt') }}-
-            ${{ runner.os }}-pkg-deps-
-            ${{ runner.os }}-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install -e .
-          pip install -r .github/workflows/requirements.txt
-
-      - name: Run unittests
-        run: |
-          coverage run -m unittest discover test/
-          coverage xml
-
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v1
-        with:
-          file: ./coverage.xml
-          flags: unittests
-          fail_ci_if_error: true
-
```

Comment:
I thought they were passing with our previous fix?

Reply:
haha yes beacause it fail due to picamera module, do you whant to see it ?

Reply:
https://github.com/pyronear/pyro-engine/pull/13/checks?check_run_id=2219038010 :)

Reply:
At the end we could remove picamera from the package because pyro-engine will not manage the capture anymore

Reply:
Alright, let's comment the test file and not the CI job then ! (otherwise we don't get a proper coverage)

Reply:
The CI won't have a camera ever so we need to find a way to test the other features but we'll do this in another PR!

Reply:
At the end the cameras will be managed via a simple script on the pi_zero. We will take this script out of the package and will no longer depend on picamera

Reply:
? what do you mean? picture taking should be done in python since we do inference with it as well, shouldn't it?
(Perhaps we need to discuss the higher purpose of this repo in an issue, I'm really getting lost here :sweat_smile: It is a refactoring but if we don't know towards what organization we refactor, we might be overlooking many things)